12LessonTwelve/homework/index.html

```diff
@@ -5,6 +5,49 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div class="container">
+    <header>
+      <link href="https://fonts.googleapis.com/css?family=Lily+Script+One" rel="stylesheet">
+      <h1>Best Selling Products</h1>
+    </header>
+    <section>
 
+      <article>
+        <div>
+          <div class="Patient Ninja">
+           <img src= http://inspirythemes.biz/html-templates/cream/images/temp-images/hoodie_1_back-470x470.jpg>
+            <div class "amount">$35.00</div>
+           <h4 class="title"><a >Patient Ninja</a></h4>
+            <input type="Submit" value="ADD TO CART">
+          </div>
+            </article>
+         <article>
+        <div>
+          <div class="Happy Ninja">
+           <img src= http://www.chameleonic.co.uk/wp-content/uploads/2015/07/hoodie_1_front-600x600.jpg>
+            <div class "amount">$35.00</div>
+           <h4 class="title"><a >Happy Ninja</a></h4>
+            <input type="Submit" value="ADD TO CART">
+          </div>
+            </article>
+            <article>
+        <div>
+          <div class="Flying Ninja">
+           <img src= http://captui.vn/wp-content/uploads/2013/06/poster_1_up-350x453.jpg>
+            <div class "amount">$12.00</div>
+            <div class "sale">SALE</div>
+           <h4 class="title"><a >Flying Ninja</a></h4>
+            <input type="Submit" value="ADD TO CART">
+          </div>
+            </article>
+              <article>
+        <div>
+          <div class="Hapy Ninja 2">
+           <img src= http://demo.themegrill.com/ample-pro/wp-content/uploads/sites/22/2013/06/poster_2_up.jpg>
+            <div class "amount">$18.00</div>
+           <h4 class="title"><a >Happy Ninja 2</a></h4>
+            <input type="Submit" value="ADD TO CART">
+          </div>
+            </article>
```

Comment:
Remember to use consistent indentation throughout your files. Atom will do it for you too! Select-all in your document (command+a), then go to Edit/Lines and choose "Auto Indent" :)